api/v1alpha1/virtualmachine_conversion.go

```diff
@@ -441,7 +449,7 @@ func convert_v1alpha2_VirtualMachineAdvancedSpec_To_v1alpha1_VirtualMachineAdvan
 }
 
 func convert_v1alpha2_BootDiskCapacity_To_v1alpha1_VirtualMachineVolume(capacity *resource.Quantity) *VirtualMachineVolume {
-	if capacity == nil || capacity.IsZero() {
+	if capacity == nil {
```

Comment:
Just curious why this change was made? Not opposed to it, just curious.

Reply:
The check was causing fuzzer failures when the capacity was zero. I think with this change the conversion is closer to a truer conversion.